doc/RASPBERRY-PI.md

```diff
@@ -0,0 +1,21 @@
+# Raspberry Pi 4 (raspberrypi4-64)
+
+- **Product Page:** https://www.raspberrypi.org/
+- **BSP (dynamic layer):** [meta-raspberrypi](https://github.com/agherzan/meta-raspberrypi.git)
+- **UART:** */dev/ttyS0* is reserved for system console in OMNECT-gateway-devel images and for output of critical kernel messages in release images.
```

Comment:
sorry for the wrong information, the serial console is here indeed free for arbitrary use in release images, I confused this with special injections for release tests.